site/content/docs/5.0/utilities/stack.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: docs
+title: Stack
+description: Use z-index utilities to quickly stack the z-index of an element.
+group: utilities
+toc: true
+---
+
+## Z-index
+
+Use z-index utilities to stack elements.
```

Comment:
I think we should explain stacking context a bit here: maybe simply adding a link to [MDN page about stacking context](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Positioning/Understanding_z_index/The_stacking_context) and mention one can use our `position` and `display` (flex related) utilities—since they are the only current utilities creating a new stacking context?

Reply:
Hey @ffoodd sound good for you? 

> Use z-index utilities to stack elements. You can combined this util with position and display utils. If you want learn more about stacking context press here

Reply:
It's better, still need a few adjustments:
1. use `position` and `display` (with back ticks since they're properties);
2. "utils": should use utilities as a full word, there's no point to abbreviate here;
3. "press here" is not an explicit label: describe what's behind the link if possible (eg. [Learn more about stacking concepts on MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Positioning/Understanding_z_index/The_stacking_context).

As an overall perspective, "You can combine" seems a bit wrong: we need to tell that `z-index` utilities won't do anything if there's no stacking context — therefore, `position`, `display` and any other property that creates a stacking context is required for it to work.

Not sure about how to word this, but I'm pretty sure another team member will pop in to help with that :)

Reply:
okass I'm going to work on that